packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -0,0 +1,274 @@
+  return {
+    chatWidget: {
+      width: "300px",
+      height: "400px",
+    },
+    chatWidgetHeader: {
+      padding: theme.spacing(1),
+      display: "flex",
+      alignItems: "center",
+      background: "#035271",
+      color: "black",
+      width: "100%",
+      height: "40px",
+    },
+    chatWidgetHeaderText: {
+      padding: theme.spacing(1),
+    },
+    iconButton: {
+      "backgroundColor": "white",
+      "border": "none",
+      "& :hover": {
+        cursor: "pointer",
+      },
+    },
+    chatWidgetContent: {
+      background: "white",
+      width: "100%",
+      height: "280px",
+      overflow: "auto",
+    },
+    chatWidgetFooter: {
+      borderTop: "1px solid #E9E9E9",
+      background: "white",
+      borderRadius: "0px 0px 12px 12px",
+      paddingTop: theme.spacing(1),
+      paddingBottom: theme.spacing(1),
+    },
+    chatWidgetTabs: {
+      minHeight: "0px",
+    },
+    chatWidgetTab: {
+      minWidth: "0px",
+      minHeight: "0px",
+      padding: "0px",
+      marginLeft: "10px",
+      marginRight: "10px",
+      fontSize: "0.6rem",
+      textTransform: "none",
+    },
+    chatWidgetInputRow: {
+      display: "flex",
+      alignItems: "center",
+      background: "white",
+      width: "100%",
+      height: "40px",
+    },
+    chatWidgetHCPToggle: {
+      height: "20px",
+    },
+    chatWidgetInput: {
+      "fontFamily": "Roboto",
+      "fontStyle": "normal",
+      "marginLeft": theme.spacing(1),
+      "display": "block",
+      "width": "90%",
+      "maxWidth": "90%",
+      "overflow": "hidden",
+      "resize": "none",
+      "height": "24px",
+      "maxHeight": "60px",
+      "lineHeight": "20px",
+      "border": "1px solid white",
+      "& :focus": {
+        outlineColor: "#F6F6F6",
+      },
+    },
+    emojiPicker: {
+      display: "none",
+    },
+    icon: {
+      margin: theme.spacing(1),
+    },
+  };
+}, { name: "ylp-chat-widget" });
+
+export interface Message {
+  text: string,
+  isMine: boolean
+}
+
+export interface ChatWidgetProps {
+  patientId: string;
+  teamId: string;
+  userId: string;
+  userRole: string;
+}
+
+function ChatWidget(props: ChatWidgetProps): JSX.Element {
+  const { t } = useTranslation("yourloops");
+  const { patientId, userId, teamId, userRole } = props;
+  const classes = chatWidgetStyles();
+  const authHook = useAuth();
+  const [showPicker, setShowPicker] = useState(false);
+  const [inputText, setInputText] = useState("");
+  const [messages, setMessages] = useState<IMessage[]>([]);
+  const [inputTab, setInputTab] = useState(0);
+  const content = useRef<HTMLDivElement>(null);
+  const input = useRef<HTMLTextAreaElement>(null);
+  const inputRow = useRef<HTMLDivElement>(null);
+  /*retrieve for the patient the first monitoring team found (only one monitoring team is allowed)*/
+  /*TODO : get monitoring team, today the field is not present in the UI so it's taking the first team*/
```

Comment:
Is there a JIRA for this todo? I personnally prefer not having any TODO in the code, but instead, having a JIRA. But in all cases, having a JIRA is the most important so it should always be done.